### ISSUE TITLE
Cannot read property 'up' of undefined

### ISSUE THREAD

**Issue by lefnire** [HabitRPG/habitica#1472]
```
2013-09-08T00:27:27.879140+00:00 app[web.2]: TypeError: Cannot read property 'up' of undefined
2013-09-08T00:27:30.492211+00:00 app[web.3]: TypeError: Cannot read property 'up' of undefined
2013-09-08T00:27:39.103542+00:00 app[web.1]:         name: 'CastError',
```

**Comment by lefnire** [comment]
note, this is all stuff that was in the API anyway - nothing new in the rewrite, just finally getting to some API bugs

**Comment by lefnire** [comment]
fixed by #1491
